actions/class.RestQtiTests.php

```diff
@@ -39,6 +41,34 @@ public function index()
         $this->returnFailure(new \common_exception_NotImplemented('This API does not support this call.'));
     }
 
+    /**
+     * Method will return qti package encoded in base64 for delivery
+     * @throws common_Exception
+     * @throws common_exception_Error
+     * @throws common_exception_NotImplemented
+     */
+    public function exportQtiPackage()
+    {
+        if ($this->getRequestMethod() != Request::HTTP_GET) {
```

Comment:
Maybe would be safe to use strict comparison here?

Reply:
And yoda code?